nmxact/sesn/sesn_cfg.go

```diff
@@ -64,16 +64,6 @@ func (r ResourceType) String() string {
 	return resTypeMap[r]
 }
 
-func ParseResType(s string) (ResourceType, error) {
-	for r, n := range resTypeMap {
-		if s == n {
-			return r, nil
-		}
-	}
-
-	return ResourceType(0), fmt.Errorf("Unknown resource type: %s", s)
```

Comment:
nmxact/sesn/sesn_cfg.go:23:2: imported and not used: "fmt"

You'll need to remove 'fmt' import above.